Mathlib/Topology/Path.lean

```diff
@@ -3,6 +3,7 @@ Copyright (c) 2020 Patrick Massot. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Patrick Massot
 -/
+import Mathlib.Data.Nat.Log
```

Comment:
Please make this a new file, e.g. `Path/CountableConcat.lean`. There is no need to add this import to ~300 downstream files.